[16.0][OU-ADD] website_slides

nguyenvietlam0640 opened the pull request.

nguyenvietlam0640 committed:
[MIG] website_slides: migrate 15 to 16

legalsylvain commented:
/ocabot migration website_slides

<dependency>Depends on : 

- website_profile : #4177</dependency>

pedrobaeza committed:
[OU-FIX+IMP] website_slides: Complete migration scripts

- Better explanations on work file
- Rework slide_type > slide_category conversion
- Shorten lines
- Fill source_type properly
- Delete record translations
- Handle nbr_article field

TT45249

pedrobaeza commented:
Push an updated branch with the following improvements:

- Better explanations on work file
- Rework slide_type > slide_category conversion
- Shorten lines
- Fill source_type properly
- Delete record translations
    
TT45249
